src/ipc/Inquirer.cc

```diff
@@ -35,15 +35,15 @@ Ipc::Inquirer::Inquirer(Request::Pointer aRequest, const StrandCoords& coords,
     AsyncJob("Ipc::Inquirer"),
     request(aRequest), strands(coords), pos(strands.begin()), timeout(aTimeout)
 {
-    debugs(54, 5, HERE);
+    debugs(54, 5, "");
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.